src/ImpactX.cpp

```diff
@@ -220,6 +226,19 @@ namespace impactx
                         transformation::CoordinateTransformation(*m_particle_container,
                                                                  CoordSystem::s);
                     }
+                    // balance particle beam in sims w/o space charge
+                    /*else if (global_np > 1)
+                    {
+                        // Resize the mesh, based on `m_particle_container` extent
+                        amrex::ParmParse pp_geometry("geometry");
+                        amrex::Vector<amrex::Real> prob_relative(max_level + 1, 1.0);
+                        pp_geometry.addarr("prob_relative", prob_relative);
+
+                        ResizeMesh();
+
+                        // Redistribute particles in the new mesh in x, y, t
+                        m_particle_container->Redistribute();
```

Comment:
Currently, this slightly looses particles over time.
We could and maybe should find a solution for this, so we can use this later on for load balancing, e.g., if we create secondaries or loose particles (e.g., to apertures/collimators).